addons/point_of_sale/tests/test_report_session.py

```diff
@@ -112,3 +112,71 @@ def test_report_session_2(self):
 
         pdf = self.env['ir.actions.report']._render_qweb_pdf('point_of_sale.sale_details_report', res_ids=session_id_2)
         self.assertTrue(pdf)
+
+    def test_report_listing(self):
```

Comment:
Hello @sbel-odoo, indeed, in existing database we need to upgrade the POS module to recreate the view.

Reply:
Oh yeah that works, but we usually avoid upgrading a whole module. Sometime we can retrieve the view only and update it, I'll check if I can find it otherwise an upgrade it is :+1: